src/fp/file/fp_file.c

```diff
@@ -233,15 +246,20 @@ struct Menu *createFileMenu(void) {
                   "thunder rage\0",
                   menuByteOptionmodProc, &pm_gCurrentSaveFile.globalBytes[0xD9]);
     menuAddStatic(&menu, 0, y, "toy box 3", 0xC0C0C0);
-    struct MenuItem *toyboxItem3Option = menuAddOption(&menu, menuX, y++,
-                                                       "pokey\0"
-                                                       "koopatrol\0"
-                                                       "super soda\0",
-                                                       menuByteOptionmodProc, &pm_gCurrentSaveFile.globalBytes[0xDA]);
+    menuAddOption(&menu, menuX, y++,
+                  "pokey\0"
+                  "koopatrol\0"
+                  "super soda\0",
+                  menuByteOptionmodProc, &pm_gCurrentSaveFile.globalBytes[0xDA]);
+    menuAddStatic(&menu, 0, y, "audio mode", 0xC0C0C0);
+    struct MenuItem *audioOption = menuAddOption(&menu, menuX, y++,
+                                                 "mono\0"
+                                                 "stereo\0",
```

Comment:
The order of the options should be switched so mono sets useMonoSound to 1.

Reply:
ah, you're right. I had it that way at first, but I think I switched it when I was trying to figure out why it wasn't changing at one point and forgot to switch it back.